tests/tests.py

```diff
@@ -80,6 +81,39 @@ def test_correct_templates_used(self):
         for url, template in urls:
             self.assertTemplateUsed(self.client.get(url), template)
 
+    def test_get_event_or_404_with_uuid_in_db(self):
+        """Check that an event is returned when an existing UUID is given."""
+        uuid = str(create_event().id)
+        event = views.get_event_or_404(uuid)
+        self.assertIsInstance(event, Event)
+
+    def test_get_event_or_404_with_uuid_not_in_db(self):
+        """Check that the function returns 404 for UUID not in db."""
+        uuid = 'd7768443-04e2-45d2-b71f-2b716bf13f13'
+        with self.assertRaises(Http404):
+            views.get_event_or_404(uuid)
+
+    def test_get_event_or_404_with_invalid_uuid(self):
+        """Check that the function returns 404 for invalid UUID."""
+        non_uuids = (
```

Comment:
There is an explanation for everything. The explanation for this particular thing is that I simply didn't think about it lol. I'll make the change and commit.